stdlib/account.go

```diff
@@ -1900,14 +1899,9 @@ func AccountContractsBorrow(
 
 	contractLocation := common.NewAddressLocation(invocationContext, address, name)
 
-	contractValue, err := invocationContext.GetContractValue(contractLocation)
-	if err != nil {
-		var notDeclaredErr interpreter.NotDeclaredError
-		if goerrors.As(err, &notDeclaredErr) {
-			return interpreter.Nil
-		}
```

Comment:
This implicitly depended on the interpreter accessing the global and returning `NotDeclaredError`. Given getting the contract value is implemented differently for the VM, remove this implicit assumption.